src/app/components/task/EditTaskDialog.js

```diff
@@ -209,21 +210,22 @@ class EditTaskDialog extends React.Component {
     const canRemove = this.state.options.length > 2;
 
     return (
-      <div style={{ marginTop: units(2) }}>
+      <Box mt={units(2)}>
```

Comment:
Please make it `mt={2}`. In general, remove all `units()` calls when using `Box`.